roles/pulp/README.md

```diff
@@ -33,15 +39,22 @@ Role Variables
       pulp-four:
         prereq_role: "pulp.pulp_three_role" # role to run immediately before the venv is created
     ```
+* `pulp_install_source`: Whether to install from `pip` (PyPI, python packages) or the Linux distro's
+  (RPM) `packages`.
+  Defaults to "pip".
+* `pulp_install_api_service`: Whether to create systemd service files for
+  pulpcore-api. Defaults to "true".
 * `pulp_cache_dir`: Location of Pulp cache. Defaults to "/var/lib/pulp/tmp".
 * `pulp_config_dir`: Directory which will contain Pulp configuration files.
   Defaults to "/etc/pulp".
 * `pulp_default_admin_password`: Initial password for the Pulp admin. **Required**.
 * `pulp_install_dir`: Location of a virtual environment for Pulp and its Python
-  dependencies. Defaults to "/usr/local/lib/pulp".
+  dependencies. Defaults to "/usr/local/lib/pulp" when `pulp_install_source==pip`.
+  Defaults to "/usr"  when `pulp_install_source==packages`.
```

Comment:
why do we have this difference between sources?

Reply:
The jinja2 templates for the systemd unit files must reference the absolute path to rq, gunicorn, etc. For example:
`ExecStart={{ pulp_install_dir }}/bin/rq worker`

Also, 3rd party RPMs could use another prefix (/usr is the default build-time prefix on Linux) hypothetically.